diagrams/Infrastructure.xml

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2021-05-23T19:14:48.150Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36" etag="HW4q6sJv9N9wXoB4X_l2" version="14.4.9" type="device"><diagram id="AH3mproc8VqpOoZvDPW4" name="Page-1">7Vrdcto8EH0aLmH8D1ySkLSdaTtMky9trxhhC6PGWK4sAvTpP8mWjC0Z7Pw4pNPATGKtZFnaPdo9u6ZnX653HwhIVl9wAKOeZQS7nj3tWZY1Mhz2j0v2ucQ0bCOXhAQFQnYQ3KA/UA4U0g0KYFoZSDGOKEqqQh/HMfRpRQYIwdvqsCWOqk9NQAg1wY0PIl36HQV0lUtH1vAg/whRuJJPNr1x3rMGcrDYSboCAd6WRPZVz74kGNP8ar27hBHXntRLft/1kd5iYQTGtM0Nd/h+/3N7M/z6u//flXs1/PHdW/RNMc0DiDZixz3Li9iEFwt2EfKLDxiHTB18+ryHPaPovIzwJmB9swjQJSZrsVe6lwpcoii6xBEmWdO+9viX3ZhSgu+h7IlxDLlQ6shgjZCAALHNKWOWOKYCJ6bD2iBCYcwaEVzyBaUJ8FEc8l5DjC49fmjyL5OPr/iXXTxAQhGz90RMQ3FymOWWN6Z9Rz5WbMs4jPicPXXqcNGKriP+3ExFmVbZ3HB31F5mgQJ2fiBeQ0r2bIi4YSRwI07OyBXt7QGGpjUUwlUJg64hhEBgPyzmPsCDXQiEPAIto2aw3M3Y1EafH3m4BBvWWQMaDSQEb+IABkK12xWi8Ibpl/dumWOp6lZBVADgaOlriGI9nj+CiyXricACRjOcIoowt7HPTABJyfiflQE5CCSytOESKwtMKV63NfeJA3gUBJ5dRcFYB4Ht1GDAcrvCgGnVgEAxJzvGCb9c70IeFgahn1iDMHMjc587jHki3YVmzuc7iJc3Rknbll2jbKFrAtmu0EM1ctQZQDxhhhFbTmFrq2pqU7EgXi5TSDX7Fct8ukldzaLSq3/bxOzvDSQPiB1G1czHz6Rrjpxr+5gR613uow9pcfq0c1r13p4SNEzFm8t2af3j7JPjTUfyCu4++XwJFwnJL6Y5qglXV0eewFE8gWUbg6GreQPPqwkIo4HblTfwNOxwtEASwTRl8jwcTHw/a77j5zR+Yki3mNx3F02MN4mhYXNEgXEw4Ww+UxVIU+RXOQHcIfqDh4XByHJF+2fWlq3pTkSNrLEvNWaQILYTbvlcFrNd5ZOZQ1cKSrPx5mG6rLUvt9QJ893AQMs0WlmT6QFviA+bHTgFJIQnJ/Qa45tbSyhfJL65SoBjbKZAn5wl36q4sZzMqHMNlWA51OfK1aHN9VJR09aJEBB+zri7nOkQZvbPXFAVuY0kc42CgM9xQWCK/oBFkYEkfGfZXt2Lnjs9xXxEdixu7hVUoxmCw5MOpW8MDM+wnweP7gmO6bTgrGr2EGSfMuc0i4AkSwFccshdMnMuUhxtKJwQX8YLLi1a1pmzBbMuW/A6SxZ0anmHCN0AdifzRXaWos8IegCUFxgE7VQsE4B0VeSGVSQrWfejWETb0PyQL3ie5MvMExg+ABAqnqtSD5Kr9ijzKI60Urs4evSrxQb3BOEoE5VnYs2px5rAljHQiYPj6Niyxy8TPCrA7ptjlY506D30ssfH21vm4K9vmPQzBjxTYqYEsc8c+DvJbSC5EVPYfJGpi9fpuvKFpso23gbXHWtYKudLophGZKHsq0wIjKs4SMSJ+MAiTvKOs3MnU9bwLQLM0gv6S0y2gHAnRbHOSpsSq5dOhl4yU+t1nVjJ49qYWbn1UHmdxMr2FCSqxd+2WZVjKhOpbxI6TqkkLS+BN0CEv9Z7Gnb/6aKApC3NVYHxOcFrqXXNp4JXfWH26uA9Xkb/NPnyHq+b4jUC685idfGOTL5MfeVQDf/svv28nf8m1t3tdBlG979+Xbd7k/baVYnW/kez0omqhKJ7TfHmqKOiRL3ebU3vnwK2aUT3/QljSrwSMSN4t9esceZKBBKrnAO+ynmSrfHvKkSU4XX6UDTD658rRMhIWQKuv4I+Tw/RMtvZ7w1Ms5Lqhq4wYdrXq2ntSVOVMpneG6Tno3MyF63CYD2RuZhu1UOajvWqzMXWc8ZNyr1ggiPk626wLYTM3iM491vm1yc9VSO/ts6aHKq02FRpcVuU2gpRV3l61/Raj9oTn2K94Cqj6WYd5QNakOB6Ul2CNN7QCMUscMtfmR4g9qwf+fUd1Tr11NTuiCDVq7rF67KnhpG/I/NuG4AkKJtz7PNGKk9H2dOcgGWqM9mv7Ab0AvqK0qTMflSk/o3v3a0jL992kvMa7th5Hjaew1dZ8/DD9Xz44ff/9tX/</diagram></mxfile>
```

Comment:
i won't be able to maintain this. so it would be better if you can remove the diagrams (or just do an ASCII diagram).